autoscheduler/frontend/src/components/App/App.tsx

```diff
@@ -9,9 +10,15 @@ import LandingPage from '../LandingPage/LandingPage';
 import SchedulingPage from '../SchedulingPage/SchedulingPage';
 import UnknownRoutePage from '../UnknownRoutePage/UnknownRoutePage';
 
-const App: React.SFC = function App() {
+const App: React.FC = function App() {
+  React.useEffect(() => {
+    const takeFullHeight = (): void => { document.body.style.height = `${window.innerHeight}px`; };
+    window.onresize = takeFullHeight;
+    return (): void => { window.removeEventListener('resize', takeFullHeight); };
+  }, []);
```

Comment:
I'd add a comment describing this briefly cause I'm mostly clueless on what this does

Reply:
I just ended up removing it because it doesn't actually help. It was basically a homemade implementation of Div100vh